draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1219,7 +1220,8 @@ before creating a new UUID. Distributed nodes are discussed in
 {{distributed_shared_knowledge}}.
 
 Implementations SHOULD choose one method for single-node UUID implementations
-that require batch UUID creation.
+that require batch UUID creation, or are otherwise concerned about monotonicity
+with high frequency UUID generation.
```

Comment:
Batch creation is an important use case but really this section applies to any code that is concerned about creating a whole bunch of UUIDs on a single node, so I think the additional wording helps with that.